<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>625</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Shortcut-Box</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>611</width>
      <height>421</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Folder</string>
     </attribute>
     <widget class="QScrollArea" name="scrollArea">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>591</width>
        <height>391</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>589</width>
         <height>389</height>
        </rect>
       </property>
       <widget class="QWidget" name="gridLayoutWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>591</width>
          <height>371</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QPushButton" name="folderBtn1">
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="acceptDrops">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string extracomment="this is added Folder">this is added Folder</string>
           </property>
           <property name="whatsThis">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Open Folder E:/tutorials&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="text">
            <string>Folder Name</string>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <property name="autoRepeat">
            <bool>false</bool>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Bash</string>
     </attribute>
     <widget class="QWidget" name="gridLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>601</width>
        <height>371</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="horizontalSpacing">
        <number>6</number>
       </property>
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Bash Button</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Scripts</string>
     </attribute>
     <widget class="QWidget" name="gridLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>601</width>
        <height>371</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>Exe Button</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>2</width>
      <height>2</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout"/>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>625</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuShortCuts">
    <property name="title">
     <string>ShortCuts</string>
    </property>
    <addaction name="actionAdd_Shortcut_Folder"/>
    <addaction name="action_Shortcut_Application"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionReload"/>
    <addaction name="actionSave_Settings"/>
    <addaction name="actionSave_Credentials"/>
    <addaction name="actionShortCuts"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionQuick_Folders"/>
    <addaction name="actionQuick_Bash"/>
    <addaction name="actionQuick_Application"/>
   </widget>
   <addaction name="menuShortCuts"/>
   <addaction name="menuView"/>
   <addaction name="menuSettings"/>
  </widget>
  <action name="actionAdd_Shortcut_Folder">
   <property name="text">
    <string>+ Shortcut Folder</string>
   </property>
  </action>
  <action name="action_Shortcut_Application">
   <property name="text">
    <string>+ Shortcut Application</string>
   </property>
  </action>
  <action name="action_Shortcut_WebLink">
   <property name="text">
    <string>+ Shortcut WebLink</string>
   </property>
  </action>
  <action name="action_Shortcut_SSO_WebLink">
   <property name="text">
    <string>+ Shortcut SSO WebLink</string>
   </property>
  </action>
  <action name="actionReload">
   <property name="text">
    <string>Reload</string>
   </property>
  </action>
  <action name="actionSave_Settings">
   <property name="text">
    <string>Save Settings</string>
   </property>
  </action>
  <action name="actionShortCuts">
   <property name="text">
    <string>ShortCuts</string>
   </property>
  </action>
  <action name="actionQuick_Folders">
   <property name="text">
    <string>Quick Folders</string>
   </property>
  </action>
  <action name="actionQuick_Bash">
   <property name="text">
    <string>Quick Links</string>
   </property>
  </action>
  <action name="actionQuick_Application">
   <property name="text">
    <string>Quick Applications</string>
   </property>
  </action>
  <action name="actionSave_Credentials">
   <property name="text">
    <string>Save Credentials</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
